src/components/Blog/CTAs/InstructionsAccordion/InstructionsAccordionItem.tsx

```diff
@@ -21,7 +21,6 @@ interface InstructionsAccordionItemProps extends InstructionItemProps {
 // Function to parse links within the title
 const parseTitle = (title: string, link: { label: string; url: string }) => {
   // Replace <LINK> with anchor tag
```

Comment:
What  about this todo

Reply:
This is something related to the blog. We have prepared this component but it´s not being used, yet. Don´t mind that one as of now, please.